meetings/summit-nov-2021

```diff
@@ -0,0 +1,254 @@
+# Next-10 Mini-Summit 18 Nov 2021
```

Comment:
I think you're missing the `.md` suffix on this file

Reply:
Thanks, fixed.